### PR TITLE
Fix disabling rules and removing rulesets (Rule board management), update release notes

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-code/findings.md [hunk 1]
```diff
@@ -228,9 +228,9 @@ To **fix a finding**:
 
 1. Update, or refactor the code such that the Semgrep rule pattern no longer matches the code.
 
-#### Removing a rule
+#### Removing a rulesets
```
Comment (armchairlinguist) [inline docs/semgrep-code/findings.md:231]:
```suggestion
#### Removing a ruleset
```

**Review by pabloest — APPROVED** [review]

File: docs/semgrep-code/findings.md (hunk 1 above)
Comment (adamkvitek) [reply docs/semgrep-code/findings.md:231]: @armchairlinguist maybe this is a matter of taste, I was just thinking that we should keep this in the plural, as we say: "Disabling rules" (Also in the plural.) Your suggestion might make it easier to search maybe?

Comment (adamkvitek) [reply docs/semgrep-code/findings.md:231]: I can also change Disabling rules to Disabling rule... let me know @armchairlinguist. :-)

Comment (armchairlinguist) [reply docs/semgrep-code/findings.md:231]: "Removing rulesets" is fine but you can't mix the plural with the singular article :)

Comment (adamkvitek) [reply docs/semgrep-code/findings.md:231]: Oh of course! :D I forgot it there... last minutes of work before past time... what a shame. :D @armchairlinguist

Comment (adamkvitek) [inline docs/semgrep-code/findings.md:231]:
```suggestion
#### Removing rulesets
```

**Pull request merged**